yarn-project/aztec-node/src/aztec-node/http_rpc_server.ts

```diff
@@ -13,11 +24,23 @@ import { AztecNode, ContractData, ExtendedContractData, L2Block, L2BlockL2Logs,
 export function createAztecNodeRpcServer(node: AztecNode) {
   const rpc = new JsonRpcServer(
     node,
-    { AztecAddress, EthAddress, ExtendedContractData, ContractData, Fr, HistoricBlockData, L2Block, L2Tx, TxHash },
+    {
+      AztecAddress,
+      EthAddress,
+      ExtendedContractData,
+      ContractData,
+      Fr,
+      HistoricBlockData,
+      L2Block,
+      L2Tx,
+      TxHash,
+      SiblingPath,
+      L1ToL2MessageAndIndex,
+    },
     { Tx, L2BlockL2Logs },
     false,
     // disable methods not part of the AztecNode interface
-    ['start', 'stop', 'getDataTreePath', 'getL1ToL2MessageAndIndex', 'getL1ToL2MessagesTreePath'],
```

Comment:
Any reason you're removing those from the list here? I thought they were added because they're not part of the `AztecNode` interface so we want to hide them

Reply:
They are actually a part of that interface because AztecNode extends StateInfoProvider. I am not sure why it was disabled.